### PR TITLE
Fix issue 285: add the limitation of participants and audiences

### PR DESCRIPTION
**Pull request by fuguigui**

### CODE REVIEW

**Comment by fuguigui** [comment]
in the application/config/constants.php: set the RDB_QUOTA as our limitation, like 75, then when the actual participants amount is lower than 75, such as 74, a new school can register new participants even if the new total number is over 75. Only if there are already participants over our limitation, a new school is unable to register.
Limitation of audience number: codes are like the rdb_quota_per_school. Now, set the limitation as 3 for each school (changable).

File: application/config/constants.php [hunk 1]
```diff
@@ -185,7 +185,7 @@
 /*
  * Date and Time Settings
  */
-$SIGN_UP_DEADLINE = '2017-04-16';
+$SIGN_UP_DEADLINE = '2018-04-16';
```
Comment (JeldorPKU) [inline application/config/constants.php]: 更改DDL不在这个PR的scope之内，你直接改回去好了，记得一个PR只做一件事。

File: application/views/registration_individual.php [hunk 2]
```diff
@@ -24,7 +24,7 @@
             <div class="col-sm-1">
                 <select class="form-control" name="ifrace">
                     <option value="0">观赛</option>
-                    <option value="1">参赛</option>
+                    <option value="1" selected="selected">参赛</option>
```
Comment (JeldorPKU) [inline application/views/registration_individual.php]: 这个更改是默认参赛的意思？

File: assets/js/race.js [hunk 3]
```diff
@@ -352,7 +352,7 @@ function resetIndividual() {
     form.find("[name='accommodation']").val("0");
     form.find("[name='gender']").val("1");
     form.find("[name='race']").val("0");
-    form.find("[name='ifrace']").val("0");
+    form.find("[name='ifrace']").val("1");
```
Comment (JeldorPKU) [inline assets/js/race.js]: 这个是以前的老bug还是什么情况，为什么这么改～

File: application/controllers/registration.php [hunk 4]
```diff
@@ -63,10 +63,10 @@ public function individual() {
             $key_set = array();
             if (!$ind_post) exit(err_msg('999'));
             $rdb_count = 0;
-            $audience_count = 0;
+            $aud_count = 0;
```
Comment (JeldorPKU) [inline application/controllers/registration.php]: 唔这里为什么要改变量的名字呢？

File: application/views/registration_individual.php (hunk 2 above)
Comment (fuguigui) [reply application/views/registration_individual.php]: 嗯嗯，我觉得参赛的人比观赛的人多

File: assets/js/race.js (hunk 3 above)
Comment (fuguigui) [reply assets/js/race.js]: 脑洞。。想把表格改成默认参赛的形式

File: application/controllers/registration.php (hunk 4 above)
Comment (fuguigui) [reply application/controllers/registration.php]: 强迫症，想要名字统一格式，都用三个字母的（捂脸）

**Pull request closed without merging**